tslib/src/pupil-api/services/redis.service.ts

```diff
@@ -25,7 +25,7 @@ export interface IRedisService {
    * @param {Array<string>} keys an array of keys to invalidate
    * @returns {Promise<void>}
```

Comment:
The jsdoc type and the typescript types are misaligned.  You can probably delete the jsdoc for *.ts files.